layouts/joomla/edit/global.php

```diff
@@ -42,7 +42,7 @@
     'version_note',
 ];
 
-$hiddenFields = $displayData->hidden_fields ?? [];
+$hiddenFields   = $displayData->get('hidden_fields') ?: [];
```

Comment:
```suggestion
$hiddenFields = $displayData->get('hidden_fields') ?: [];
```
Remove unnecessary spaces.